articles/getting-started-with-quasar-framework/index.md

```diff
@@ -0,0 +1,295 @@
+layout: engineering-education
+status: 
+published: 
+url: /engineering-education/getting-started-with-quasar-framework/
+title: Getting Started with Quasar Framework
+description: In this article, we will look at get started with quasar framework, its components structure, its pros and finally cons.
+author: espira-marvin
+date: 
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/getting-started-with-quasar-framework/hero.jpg
+    alt: Getting Started with Quasar
+---
+
+### Introduction
+
+Quasar Framework is a Vue.js based framework used to develop cross-platform applications using one codebase saving development costs. Quasar offers a state of art user Interface and support builds for:
+  - single-page applications 
+  - progressive web applications
+  - server-side rendering
+  - mobile apps (Ios and android) using cordova or capacitor 
+  - multi-platform desktop apps using electron 
+  - browser extensions. 
+  
+The overhead of using quasar is minimal as it implements the best practices in web development out-of-the-box, taking into account performance and responsiveness in mind. It also has components for almost every web development need.
+
+### Prerequisites
+
+1. Knowledge of Vue js
+2. Quasar CLI
+3. Node.js 10.x or newer excluding 13 and 15
+4. Npm 5.10 or newer / yarn 1.2 or newer
+
+### Installing Quasar Framework
+There are three ways we can install quasar as listed below. In this tutorial, we're going to use the third method, `using quasar CLI` which comes with all quasar features out-of-the-box. You can skip the first two and try them at a later time.
+
+1. Embedding into an existing project through CDN as shown below.
+
+```html
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <!-- Add  at the end of your body tag -->  
+    <script src="https://cdn.jsdelivr.net/npm/vue@^2.0.0/dist/vue.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/quasar@1.15.6/dist/quasar.umd.min.js"></script>
+    
+    <script>
+      new Vue({
+        el: '#q-app',
+        data: function () {
+          return {}
+        },
+        methods: {},
+      })
+    </script>
+  </body>
+</html>
+```
+In your body tag, you should use Quasar components in a div with and id `q-app` as shown below:
+```html
+ <div id="q-app"> ... </div>
+ ```
+Do not use self-closing tags like `<q-list />` instead, use `<q-list></q-list>`
+
+Note this is not the best practice, since you will have to make sure to match all quasar related tags to the same version you have installed. In the script, we have used version `@1.15.6`.
+
```

Comment:
Please use consistent capitalization of words like 'Quasar', etc. Review your document correcting this.